### PR TITLE
add guide for tryInvoke deprecation

### PR DESCRIPTION
**Pull request by locks**
Adds guide for [tryInvoke deprecation](https://emberjs.github.io/rfcs/0674-deprecate-transition-methods-of-controller-and-route.html).

I adjusted the text from the RFC to something that felt smoother, please review!

### CODE REVIEW

File: content/ember/v3/try-invoke.md
Comment (ijlee2) [inline content/ember/v3/try-invoke.md:10]:
```suggestion
In most cases, function arguments should not be optional, but in the rare occasion that an argument is optional by design, we can replace `tryInvoke` with JavaScript's [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).
```

File: jsconfig.json
Comment (ijlee2) [inline jsconfig.json:1]: Can you leave a note in the pull request why this file was added?

**Review by ijlee2 — APPROVED** [review]
There's a word missing that needs to be added before merging the PR. Thanks for working on updating the `deprecation-app`!

File: jsconfig.json
Comment (locks) [reply jsconfig.json:1]: Ugh, it shouldn't have been. I'll configure my global git ignore!

**Comment by locks** [comment]
Thank you for the swift review! Will merge once CI passes after the latest changes.

**Pull request merged**
